docs/site/tutorials/device_placement.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>For consistency with other Swift for TensorFlow tutorials, please only capitalize the first letter in a sentence in headings.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>